admin_guide/install/docker_registry.adoc

```diff
@@ -38,6 +47,27 @@ To see a full list of options that you can specify when creating the registry:
 $ oadm registry --help
 ----
 
+*Storage for the Registry*
+
+The registry stores Docker images and metadata. If you simply deploy a pod with
+the registry, it uses an ephemeral volume that is destroyed if the pod exits.
+Any images anyone has built or pushed into the registry would disappear.
+
+Using the `--mount-host` option, you can specify a directory for the registry to
```

Comment:
`--mount-host` will mount a directory from the node on which the registry container lives. If you scale up the `docker-registry` deployment config, it's possible that your registry pods and containers will run on different nodes. This means that you could end up with 2 or more registry containers, each with its own local storage. This is Not Good™ and will lead to unpredictable behavior, as subsequent requests to pull the same image again and again may succeed some times and fail others, depending on which container the request ultimately goes to.

Also, `--mount-host` requires that the registry container run in privileged mode. This is automatically enabled for you when you specify `--mount-host`. However, by default, not all pods are allowed to run privileged containers. If you still want to use this option, you'll need to do something like this:

1. Create a service account for the registry to run as: `echo '{"kind":"ServiceAccount","apiVersion":"v1","metadata":{"name":"registry"}}' | oc create -f -`. Here the service is named `registry` but you could choose any name you'd like.
2. Add the service account to the list of users allowed to run privileged containers: `oc edit scc privileged` (add a line under `users` with the user name `system:serviceaccount:default:registry`).
3. Create the registry and specify it should use the service account: `oadm registry --service-account=registry ...`